lib/src/browser/disposable_browser.dart

```diff
@@ -198,14 +199,20 @@ class Disposable implements disposable_common.Disposable {
   void manageDisposable(disposable_common.Disposable disposable) =>
       _disposable.manageDisposable(disposable);
 
+  @deprecated
   @override
   void manageDisposer(disposable_common.Disposer disposer) =>
```

Comment:
It's [deprecated there too](https://github.com/Workiva/w_common/pull/47/files#diff-3758fc2d32c5ee729b97395b0dd3e885R63). I found if it was not deprecated in both places I woulden't get analysis warnings in my IDE.